Realm/ObjectStore/list.hpp

```diff
@@ -106,6 +106,8 @@ class List {
         size_t valid_count;
     };
 
+    LinkViewRef get_linkview() const { return m_link_view; }
```

Comment:
It'd be nice to continue to avoid exposing the LinkView directly. Could maybe just make List/Result be able to produce an AnyThreadConfined?